ocean/global_ocean/WC_SO_Atl10/files_for_e3sm/config_files_for_e3sm.ini

```diff
@@ -0,0 +1,78 @@
+# This is the custom config file specific to this test case.  Any values here
+# will override the defaults in defaults.ini  Users wishing to alter the
+# configuration should modify this file, rather than defaults.ini
+
+[main]
+nprocs = 1
+# the file name of the initial condition, possibly after spin-up
+initial_condition = ../../spin_up/simulation/restarts/restart.0001-01-21_00.00.00.nc
+
+[mesh]
+short_name = ${prefix}${min_res}to${max_res}E${e3sm_version}r${mesh_version}
+long_name = ${prefix}${min_res}to${max_res}kmL${levels}E3SMv${e3sm_version}r${mesh_version}
+prefix = WC_SO_Atl
```

Comment:
We likely want a different name.  Just `WC`?

Reply:
I'm thinking WC_Atl would be a reasonable choice.  We may want Atl as just having the resolution min/max could be misleading.

Reply:
Done!

Reply:
I think WCAtl (without any underscores) would be better.  I think I recall there were problems with punctuation in mesh names (possibly including underscores).

Reply:
WCAtl sounds good to me.